src/Cryptol/Parser/Unlit.hs

```diff
@@ -134,6 +137,34 @@ latex = comment []
   isBeginCode l = "\\begin{code}" `Text.isPrefixOf` l
   isEndCode l   = "\\end{code}"   `Text.isPrefixOf` l
 
-
+rst :: [Text] -> [Block]
+rst = comment []
+  where
+  isBeginCode l = case filter (not . Text.null) (Text.splitOn " " l) of
```

Comment:
I wonder if this should use `Text.split isSpace` instead, both for consistency with `isEmpty`/`isCode` and to catch other whitespace characters.

Reply:
Good idea, will change.